src/main/kotlin/org/jetbrains/research/ml/ast/transformations/deadcode/DeadCodeRemovalTransformation.kt

```diff
@@ -0,0 +1,62 @@
+package org.jetbrains.research.ml.ast.transformations.deadcode
```

Comment:
Please, add your copyright to all source code files you wrote

Reply:
I think we should change the copyright in the `LICENSE` into `Copyright (c) 2020 Birillo A., Lyulina E., Bobrov N`., and use this one in all new files including files from this PR